resources/definitions/koonovo_kn3.def.json

```diff
@@ -0,0 +1,160 @@
+{
+    "name": "KOONOVO KN3 IDEX",

```

Comment:
I was unable to find the KN3 or KN5 online. Are you sure the names are correct? Do you have a spec sheet anywhere?